dioptas/widgets/plot_widgets/ColormapPopup.py

```diff
@@ -87,11 +96,34 @@ def __init__(self, parent=None):
         self._maxEdit.editingFinished.connect(self._rangeChanged)
         layout.addRow('Max:', self._maxEdit)
 
-        self._autoscaleButton = QtWidgets.QPushButton('Autoscale', self)
-        self._autoscaleButton.clicked.connect(self._autoscaleButtonClicked)
+        reloadIcon = QtWidgets.QApplication.instance().style().standardIcon(
+            QtWidgets.QStyle.SP_BrowserReload
+        )
+        self._autoscaleButton = QtWidgets.QPushButton(reloadIcon, "Reset", self)
+        self._autoscaleButton.setToolTip("Scale colormap range with current mode")
+        self._autoscaleButton.clicked.connect(self._autoscaleRequested)
         self._autoscaleButton.setAutoDefault(False)
         self._autoscaleButton.setEnabled(False)
-        layout.addRow('', self._autoscaleButton)
+        layout.addRow("", self._autoscaleButton)
+
+        self._resetButtonGroup = QtWidgets.QButtonGroup(self)
+        resetModesLayout = QtWidgets.QVBoxLayout()
+        resetModesLayout.setContentsMargins(0, 0, 0, 0)
+        for text, (mode, tooltip) in self._RESET_MODES.items():
+            radioButton = QtWidgets.QRadioButton(text, self)
+            radioButton.setToolTip(tooltip)
+            radioButton.setChecked(mode == utils.auto_level.mode)
```

Comment:
Switched to radio buttons.
I hesitated to remove the "Reset" button and provide a button for each mode since clicking on a radio button applies the autoscale... This would make it more compact, but having the radio buttons make the current mode explicit